src/com/sap/piper/integration/TransportManagementService.groovy

```diff
@@ -332,8 +258,15 @@ class TransportManagementService implements Serializable {
             echo("Response content '${response.content}'.")
         }
         
+        // because the API is called with params, the return is always a map with either a empty list or a list containing one element
+        Map mtaExtDescriptor = [:]
```

Comment:
Is there a specific reason behind returning empty map (and not null or undefined), if no extension descriptor is found?

Reply:
It would be convenient for the following check in tmsUpload and I also try to avoid null and undefined cases.